sdk/spring/azure-spring-boot-starter-active-directory/README.md

```diff
@@ -214,12 +214,7 @@ spring.security.oauth2.client.provider.azure.jwk-set-uri=https://login.microsoft
 spring.security.oauth2.client.registration.azure.scope=openid, https://graph.windows.net/user.read, {your-customized-scope}
```

Comment:
configuration items like `spring.security.oauth2.client.registration.azure.xxx` is not supported, it's OK to fix in another PR.